cupyx/jit/_cuda_types.py

```diff
@@ -253,6 +275,29 @@ def __str__(self) -> str:
         return f'{self.child_type}*'
 
 
+class ContiguousArray(PointerBase):
+    # used to define ndarray of local memory, in <dtype>[3][4] format
```

Comment:
Could you fix class name and comments?
```suggestion
class SizedArray(PointerBase):
    # used to define C-array type, in <type>[dim0][dim1] format
```